### PR TITLE
fix windows php8 php_stdint.h path

### PR DESCRIPTION
**Pull request by lifenglsf**
fix windows php8 php_stdint.h path

### CODE REVIEW

File: php_memcached_private.h [hunk 1]
```diff
@@ -48,7 +48,11 @@
 #include <ext/standard/basic_functions.h>
 
 #ifdef PHP_WIN32
+#if PHP_VERSION_ID >= 80000
```
Comment (sodabrew) [inline php_memcached_private.h:51]: Please indent the directive one space

File: php_memcached_private.h [hunk 2]
```diff
@@ -48,7 +48,11 @@
...
+#include "php_stdint.h"
```
Comment (sodabrew) [inline php_memcached_private.h:52]: Please indent two spaces and add a second space to the alternate win32 include line

**Review by sodabrew — COMMENTED** [review]
I don't have a win32 test machine myself. If this works for you I'll take your word for it. Is there a release note that we can link to in the comment indicating this was a known change in PHP 8.x?

File: php_memcached_private.h (hunk 2 above)
Comment (lifenglsf) [reply php_memcached_private.h]: > I don't have a win32 test machine myself. If this works for you I'll take your word for it. Is there a release note that we can link to in the comment indicating this was a known change in PHP 8.x?

i compiled dll file
basic usage works fine

**Comment by cmb69** [comment]
> Is there a release note that we can link to in the comment indicating this was a known change in PHP 8.x?

See https://github.com/php/php-src/pull/5323. This PR is okay, but for PHP 8 I don't see the need for the platform specific code at all. `include "php_stdint.h"` should be good for all platforms.

**Comment by lifenglsf** [comment]
> > Is there a release note that we can link to in the comment indicating this was a known change in PHP 8.x?
> 
> See [php/php-src#5323](https://github.com/php/php-src/pull/5323). This PR is okay, but for PHP 8 I don't see the need for the platform specific code at all. `include "php_stdint.h"` should be good for all platforms.

include "php_stdint.h" worked for php7?

**Comment by cmb69** [comment]
> include "php_stdint.h" worked for php7?

No. Only as of PHP 8.0.0.

**Pull request merged**